libs/designsystem/src/lib/components/modal/modal-wrapper/modal-wrapper.component.ts

```diff
@@ -268,6 +275,7 @@ export class ModalWrapperComponent implements Modal, AfterViewInit, OnInit, OnDe
   @HostListener('window:ionKeyboardDidShow', ['$event'])
   _onKeyboardWillShow(event: { detail: { keyboardHeight: number } }) {
     this.keyboardVisible = true;
+    if (!event.detail) return;
```

Comment:
The event always has the `detail` property... If it's because of dispatching fake events from units tests, you should look into fixing that instead of patching this...
```suggestion
```